app/views/imports/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_with(url: podcast_imports_path(import.podcast), model: [import.podcast, import], method: :post, html: {autocomplete: "off"}) do |form| %>
+<%= form_with(url: podcast_imports_path(import.podcast), model: [import.podcast, import.becomes(PodcastImport)], method: :post, html: {autocomplete: "off"}) do |form| %>
```

Comment:
is the `becomes` method how the form can expect that the `type` of class might change?